cypress/integration/IntersectionDetector/Visibility_notification.feature

```diff
@@ -0,0 +1,13 @@
+Feature: The IntersectionDetector notifies the consumer when becoming visible
+
+    Scenario: The user scrolls the IntersectionDetector into the view
+        Given the detector is not visible
+        When the user scrolls the detector into the view
+        Then the callback passed to onIsVisible should be called
```

Comment:
The component has a prop called `onVisibilityChange` but here in the scenarios you are talking about `onIsVisible`. I found that a little confusing.

Reply:
:+1: